docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -6,38 +6,33 @@ description: Deployment and Operation of edge authorizers
 
 # Deployment and Operation
 
-Edge authorizers can be deployed in various ways based on the application and target environment needs and requirements.
-In its most basic form, an authorizer is a Docker container instance, which can be deployed as a standalone service,
-sidecar, or development workstation.
+Edge Authorizers can be deployed in various ways based on the application and target environment needs and requirements.
+In its most basic form, an authorizer is a Docker container instance, which can be deployed as a standalone microservice or a sidecar.
 
-An edge authorizer always operates in the context of a (single) Aserto organization and is assigned a specific policy
+An Edge Authorizer always operates in the context of a (single) Aserto organization and is assigned a specific policy
 instance to serve. This information is provided to the edge authorizer via a configuration file, which it uses to send a
 discovery call to the Aserto control plane.
 
-## Edge Authorizer Flavors
+<img width="70%" src="/deployment-authorizer.png" />
 
-:::danger Note
-The `sidecar` authorizer is replaced by the [topaz](https://github.com/aserto-dev/topaz).
-:::
-
-
-Currently the following flavors of edge authorizers are available:
+## Docker image
+The Authorizer ships as a docker image and can be pulled with following command:
```

Comment:
"The Edge Authorizer" or "Topaz" 

the note right underneath this sentence just uses Topaz. Trying to help ppl connect the dots. Authorizer = edge authorizer = topaz